app/src/main/java/com/monkeyteam/chimpagne/model/database/ChimpagneAccountManager.kt

```diff
@@ -198,6 +198,36 @@ class ChimpagneAccountManager(
         },
         onFailure)
   }
+
+  fun getAllOfMyEvents(
+      onSuccess: (createdEvents: List<ChimpagneEvent>, joinedEvents: List<ChimpagneEvent>) -> Unit,
+      onFailure: (Exception) -> Unit
+  ) {
+    if (database.accountManager.currentUserAccount == null) {
+      return onFailure(NotLoggedInException())
+    }
+
+    val eventIDs = database.accountManager.currentUserAccount?.joinedEvents
+    if (eventIDs == null) {
```

Comment:
useless, eventIDs cannot be null as database.accountManager.currentUserAccount isn't null

Reply:
Good catch!
I meant to do the case where the account has not registered any events nor joined any events. In retrospect I should have just checked if eventIDs were empty.